src/style.css

```diff
@@ -1 +1,124 @@
-/* add styles here */
+* {
```

Comment:
Using the universal selector '*' is not recommended as it impacts performance. Consider applying styles directly to specific elements or classes that require them.